projects/kit/src/lib/processors/with-placeholder.ts

```diff
@@ -63,8 +63,8 @@ export function maskitoWithPlaceholder(
             },
         ],
         postprocessors: [
-            ({value, selection}, initialElementState) =>
-                initialElementState.value && (focused || !focusedOnly)
```

Comment:
It is legacy code part that causes excess `Input`-dispatching.

It is not more required after closing of this issue:
* https://github.com/taiga-family/maskito/pull/778